src/network/tc/cake.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 VMware, Inc. */
+#pragma once
+
+#include "conf-parser.h"
+#include "qdisc.h"
+#include "time-util.h"
+
+typedef struct CommonApplicationsKeptEnhanced {
+        QDisc meta;
+
+        int overhead;
+        int mpu;
```

Comment:
what are the units of this? should this really be signed? signed values don#t make sense here?